LICENSE

```diff
@@ -1,202 +1,14 @@
+Copyright (c) 2016, The OpenThread Authors.
+All rights reserved.
 
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
Apologies, I got it confused with the boilerplate thing that goes in each individual file.  I've reverted, with the include of the copyright at the top and removed the appendix at the end that just had the instructions on applying the license.